modules/mesos-cluster/main.tf

```diff
@@ -7,6 +7,7 @@ variable "slave_instance_type" {}
 variable "subnet1" {}
 variable "subnet2" {}
 variable "subnet3" {}
+variable "sns_topic_arn" {}
 
```

Comment:
We just need to add one more empty variable for `victorops_cloudwatch_endpoint`.